zio-http/src/main/scala/zhttp/service/Server.scala

```diff
@@ -56,6 +58,11 @@ object Server {
     flowControl: Boolean = false,
   )
 
+  /**
+   * Holds server bootstrap information.
+   */
+  final case class Bootstrap(port: Int = 0)
```

Comment:
Bootstrap is confusing with ServerBootstrap, no? What was wrong with `Start`?

Reply:
I like `Start` as well but, to me, it doesn't convey the whole intent - if it makes sense.
Maybe we could make it `BootstrapConfig`?